contrib/coredns/policy/policy.go

```diff
@@ -149,7 +178,29 @@ func (p *PolicyMiddleware) ServeDNS(ctx context.Context, w dns.ResponseWriter, r
 	}
 
 	if response.Permit {
-		return middleware.NextOrFailure(p.Name(), p.Next, ctx, w, r)
+		status,err := middleware.NextOrFailure(p.Name(), p.Next, ctx, w, r)
+		if err != nil {
+			return status, err
+		}
+
+		responseattrs := []*pb.Attribute{&pb.Attribute{Id: "type", Type: "string", Value: "response"}}
+		if len(r.Question) > 0 {
+			q := r.Question[0]
+			responseattrs = append(attrs, &pb.Attribute{Id: "domain_name", Type: "domain", Value: strings.TrimRight(q.Name, ".")})
+
+		}
+		if foundSourceIP {
+			responseattrs = append(attrs, &pb.Attribute{Id: "proxy_source_ip", Type: "address", Value: state.IP()})
+		} else {
+			responseattrs = append(attrs, &pb.Attribute{Id: "source_ip", Type: "address", Value: state.IP()})
+		}
+		responseattrs = append(attrs, &pb.Attribute{Id: "policy_id", Type: "string", Value: "Dummy"})
+
```

Comment:
Based on the pepcli test program 
  - type: response
    policy_id: UNKNOWN
    address: 127.0.0.1

**Changed code to -**
                state := request.Request{W: w, Req: r}
		rr := r.Answer[0]
		switch state.Family() {
		case 1:
			attrs = append(attrs, &pb.Attribute{Id: "address", Type: "address", Value: rr.(*dns.A).A})
		case 2:
			attrs = append(attrs, &pb.Attribute{Id: "address", Type: "address", Value: rr.(*dns.AAAA).AAAA})

		}